src/components/App.js

```diff
@@ -0,0 +1,56 @@
+import { getUser, getFolder } from "../api";
+import "./App.css";
+import NavigationBar from "./NavigationBar";
+import { useEffect, useState } from "react";
+import FolderBar from "./FolderBar";
+import CardSection from "./CardSection";
+import Footer from "./Footer";
+
+function App() {
+  const [profile, setProfile] = useState(null);
+  const [folderName, setFolderName] = useState("");
+  const [userName, setUserName] = useState("");
+  const [profileImage, setProfileImage] = useState("");
+  const [cardList, setCardList] = useState(null);
```

Comment:
각각 데이터가 사실 다르다고 볼 수도 있기 때문에 상태를 여러개 만들어주신 것 같은데요~
함께 변경되거나, 출처가 같은(api에서 받아오는) 데이터들은 하나의 상태로 관리해주시는게 좋을 것 같아요~
아래와 같이 하나의 상태로 관리해보면 좋을 것 같아요!
```suggestion
  const [folder, setFolder] = useState(null);
```